packages/parser-babel-config/src/schema/babelConfigSchema.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
If you want to know if you have any property that doesn't exists/support in babel, you need to add the `"additionalProperties": false`